source/reference/pmm-admin.rst

```diff
@@ -0,0 +1,314 @@
+    ``--disable-tablestats-limit=DISABLE-TABLESTATS-LIMIT``  
+       Table statistics collection will be disabled if there are more than specified number of tables
+       (default: server-defined).
+    ``--environment=ENVIRONMENT``
+       Environment name.
+    ``--cluster=CLUSTER``
+       Cluster name.
+    ``--replication-set=REPLICATION-SET``  
+       Replication set name.
+    ``--custom-labels=CUSTOM-LABELS``
+       Custom user-assigned labels.
+    ``--skip-connection-check``
+       Skip connection check.
+    ``--tls``
+       Use TLS to connect to the database.
+    ``--tls-skip-verify``
+       Skip TLS certificates validation.
+
+PostgreSQL Options
+==================
+    
+``add postgresql [<flags>] [<name>] [<address>]``
+    Add PostgreSQL to monitoring.
+
+    ``--node-id=NODE-ID``
+      Node ID (default is auto-detected).
+    ``--pmm-agent-id=PMM-AGENT-ID``
+       The pmm-agent identifier which runs this instance (default is auto-detected).
+    ``--username="postgres"``
+       PostgreSQL username.
+    ``--password=PASSWORD``
+       PostgreSQL password.
+    ``--query-source=pgstatements``
+       Source of SQL queries, one of: pgstatements, none (default: pgstatements).
+    ``--environment=ENVIRONMENT``
+       Environment name.
+    ``--cluster=CLUSTER``
+       Cluster name.
+    ``--replication-set=REPLICATION-SET``  
+       Replication set name
+    ``--custom-labels=CUSTOM-LABELS``
+       Custom user-assigned labels.
+    ``--skip-connection-check``
+       Skip connection check.
+    ``--tls``
+       Use TLS to connect to the database.
+    ``--tls-skip-verify``
+       Skip TLS certificates validation.
+
+ProxySQL Options
+================
+    
+``add proxysql [<flags>] [<name>] [<address>]``
+    Add ProxySQL to monitoring.
+
+    ``--node-id=NODE-ID``
+       Node ID (default is auto-detected).
+    ``--pmm-agent-id=PMM-AGENT-ID``
+       The pmm-agent identifier which runs this instance (default is auto-detected).
+    ``--username="admin"``
+       ProxySQL username.
+    ``--password="admin"``
+       ProxySQL password.
+    ``--environment=ENVIRONMENT``
+       Environment name.
+    ``--cluster=CLUSTER``
+       Cluster name.
+    ``--replication-set=REPLICATION-SET``  
+       Replication set name.
+    ``--custom-labels=CUSTOM-LABELS``
+       Custom user-assigned labels.
+    ``--skip-connection-check``
+       Skip connection check.
+    ``--tls``
+       Use TLS to connect to the database.
+    ``--tls-skip-verify``
+       Skip TLS certificates validation.
+    
+********
+EXAMPLES
+********
+
+.. code-block:: sh
+
+   # pmm-admin status
+   Agent ID: /agent_id/c2a55ac6-a12f-4172-8850-4101237a4236
+   Node ID : /node_id/29b2cc24-3b90-4892-8d7e-4b44258d9309
+   PMM Server:
+    URL : https://x.x.x.x:443/
+    Version: 2.5.0
+   PMM Client:
+    Connected : true
+    Time drift: 2.152715ms
+    Latency : 465.658µs
+    pmm-admin version: 2.5.0
+    pmm-agent version: 2.5.0
```

Comment:
If i remember correctly this is not yet implemented?

Reply:
oh .. i see. So this PR is mix: new  page + issues from Sprint

Reply:
Yes, two birds, one stone.